Tropos/Controllers/SettingsController.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+@objc(TRSettingsController) final class SettingsController: NSObject {
+    private let locale: NSLocale
+
+    var unitSystem: UnitSystem {
+        get {
+            let rawUnitSystem = NSUserDefaults.standardUserDefaults().integerForKey(TRSettingsUnitSystemKey)
+            return UnitSystem(rawValue: rawUnitSystem)!
+        }
+        set {
+            NSUserDefaults.standardUserDefaults().setInteger(newValue.rawValue, forKey: TRSettingsUnitSystemKey)
+        }
+    }
+
+    init(locale: NSLocale) {
+        self.locale = locale
+    }
+
+    convenience override init() {
+        self.init(locale: .autoupdatingCurrentLocale())
+    }
+
+    func registerSettings() {
+        registerUnitSystem()
+        registerLastVersion()
```

Comment:
Here's a snippet from the docs:

> ... use this method only if you cannot wait for the automatic synchronization (for example, if your application is about to exit) ...

Because this is being called at application startup, the explicit `-synchronize` isn't necessary, which is why I removed it.